notebooks/Gaslim_draw.ipynb

```diff
@@ -217,7 +216,7 @@
    "outputs": [],
    "source": [
     "# After defining the function, you can use it like: \n",
-    "catalog.apply_selection_function(btk.utils.basic_selection_function, 1, 25)\n",
+    "catalog.apply_selection_function(btk.utils.basic_selection_function, 4, 27)\n",
```

Comment:
Maybe write those two last arguments with their signature (i.e. as keyword args) to make the selection more explicit.
Or name these arguments beforehand so we understand what 4 and 27 stand for.